public/app/features/dashboard-scene/conditional-rendering/hooks/useIsConditionallyHidden.tsx

```diff
@@ -14,11 +14,29 @@ function getPlaceholderConditionalRendering(): ConditionalRenderingGroup {
   return placeholderConditionalRendering;
 }
 
-export function useIsConditionallyHidden(scene: SceneObject): [boolean, string | undefined, ReactNode | null, boolean] {
-  const conditionalRenderingToRender =
-    'conditionalRendering' in scene.state && scene.state.conditionalRendering instanceof ConditionalRenderingGroup
-      ? scene.state.conditionalRendering
-      : getPlaceholderConditionalRendering();
+export function useIsConditionallyHidden(
+  scene: SceneObject,
+  itemIdx?: number
+): [boolean, string | undefined, ReactNode | null, boolean] {
+  let conditionalRenderingToRender = getPlaceholderConditionalRendering();
+
+  if (itemIdx !== undefined) {
+    if (
+      'repeatedConditionalRendering' in scene.state &&
```

Comment:
nit: could we extract this condition into a function to make it more readable? something like `isConditionalRenderingGroupItem` (or a better name 😅)